src/libcore/ptr.rs

```diff
@@ -212,6 +212,14 @@ pub unsafe fn write_volatile<T>(dst: *mut T, src: T) {
 #[lang = "const_ptr"]
 impl<T: ?Sized> *const T {
     /// Returns true if the pointer is null.
+    ///
+    /// Example:
+    ///
```

Comment:
It is, but I currently value consistency over a bit of verbosity for simple examples.